packages/labs/ssr/src/test/integration/tests/basic.ts

```diff
@@ -4698,6 +4698,178 @@ export const tests: {[name: string]: SSRTest} = {
     };
   },
 
+  'LitElement: Reflected number attribute': () => {
+    return {
+      registerElements() {
+        class LEReflectedNumberAttribute extends LitElement {
+          @property({type: Number, reflect: true})
+          num = 42;
+        }
+        customElements.define(
+          'le-reflected-number-attribute',
+          LEReflectedNumberAttribute
+        );
+      },
+      render() {
+        return html`
+          <le-reflected-number-attribute></le-reflected-number-attribute>
+        `;
+      },
+      expectations: [
+        {
+          args: [],
+          async check(assert: Chai.Assert, dom: HTMLElement) {
+            const el = dom.querySelector(
+              'le-reflected-number-attribute'
+            )! as LitElement;
+            await el.updateComplete;
```

Comment:
might be nice to eventually reduce some of the boilerplate here by treating the render() as a fixture and auto-hook-up finding an element and waiting for updateComplete, etc.